server/start_server.sh

```diff
@@ -1,10 +1,15 @@
 #!/bin/bash
 
+sqlite3_loc=/usr/bin/sqlite3
+if [ ! -f $sqlite3_loc ]; then
+  echo "sqlite3 is not installed, installing.."
+  sudo apt-get install sqlite3
+fi;
```

Comment:
I'd like this to be an execution script and not an installation script (which also depends on the OS you're running on)